galasa-managers-parent/galasa-managers-internal-parent/dev.galasa.sem.manager/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 
 description = 'Galasa Internal SEM Manager'
 
-version = '0.21.0'
+version = '0.25.0'
```

Comment:
Is there the equivalent of a release.yaml to incorporate this into the internal bom?

Reply:
I see you have updated